index.html

```diff
@@ -73,10 +73,10 @@ <h3 class="main-title style-count" id="count"></h3>
         <div class="card card__animate">
           <img class="card-img-top img-fluid" data-src="./images/cops.png" alt="Card image cap">
           <div class="card-body">
-            <h5 class="card-title">Title of Name Card</h5>
+            <h5 class="card-title">Khushal Sahni</h5>
```

Comment:
You are required to create a copy of the namecard. Not to edit the existing.

Reply:
ok I updated the pull request as per commit 79c86cd

Reply:
@jmig5776